src/formats/apple-notes/convert-note.ts

```diff
@@ -47,7 +47,10 @@ export class NoteConverter {
 				attrText += this.note.noteText.substring(offsetStart, offsetEnd);
 				
 				offsetStart = offsetEnd;
-				nextIsSame = attrEquals(attr, this.note.attributeRun[i+1]);
+				
+				if (i == this.note.attributeRun.length - 1) nextIsSame = false;
```

Comment:
Thanks for changing this. What do you think about stylistically tweaking it like this?

```
nextIsSame = (i == this.note.attributeRun.length - 1)
    ? false
    : attrEquals(attr, this.note.attributeRun[i+1]);
```